test/autests/gold_tests/body_verification/http.yaml

```diff
@@ -0,0 +1,140 @@
+meta:
+  version: '1.0'
+sessions:
+- protocol:
+  - name: http
+    version: 1.1
+  - name: tls
+    sni: test_sni
+  - name: tcp
+  - name: ip
+    version: 4
+  transactions:
+  - all:
+      headers:
+        fields:
+        - [uuid, 1]
+    client-request:
+      method: POST
+      scheme: https
```

Comment:
No need to include scheme. They're not used, and it is redundant with the `sessions:protocol:tls` node.